indico/modules/designer/controllers.py

```diff
@@ -177,8 +179,21 @@ def _process_args(self):
 
     def _process(self):
         title = f'{self.template.title} (copy)'
-        new_template = DesignerTemplate(title=title, type=self.template.type, data=self.template.data,
-                                        **self.target_dict)
+        new_template = DesignerTemplate(title=title, type=self.template.type, **self.target_dict)
+
+        data = self.template.data
+        image_items = [(i, item) for i, item in enumerate(data['items']) if item['type'] == 'fixed_image']
+        for i, image_item in image_items:
+            if image_item.get('image_id'):
```

Comment:
When would there be a `fixed_image` item without an `image_id`? If this is not expected to happen, no need to check for it - just let it fail if there's garbage data in the DB.

Reply:
This could happen for example when a user inserts a fixed_image element in the badge designer but they do not upload any image into it. Then the image_id would be null

Reply:
I think failing validation (so the user has to remove the item or add an image) would be better than storing broken data...

Reply:
Okay